Answers.md

```diff
@@ -0,0 +1,9 @@
+If I were to describe semantic HTML, I would say that semantic HTML is a way of creating div's that have meaning. For example, a <p> is a stands for paragraph. It also allows code to be more readable by users and allows for better SEO.
+
+The differences between display:block and inline is that display block takes up one whole line and inline block only encases the content thats inside it.
+
```

Comment:
Yup pretty much on it.

block - it's width is defined by how wide its container is
inline - it's width is defined by how much stuff is inside of it